triplea_maps.yaml

```diff
@@ -88,10 +88,10 @@
   version: 1
   img: http://tripleamaps.sourceforge.net/images/TripleA_total_world_war_mini.png
   description: |
-    <br><em><b><font size="6">Created by Rolf Larsson and Hepster</font></b></em>
+    <br><em><b style="font-size: 6px;">Created by Rolf Larsson and Hepster</b></em>
```

Comment:
This [post](https://stackoverflow.com/questions/20805668/what-is-best-replacement-for-font-size-x-in-css) shows the equivalents in terms of named sizes.  I confirmed that CSS size `xx-large` does indeed render the same as HTML size `6`.